### DISCUSSION TITLE
Missing and not fully working features in IntelliJ plugin

### DISCUSSION THREAD

**Discussion by DaSchTour** [nrwl/nx-console#1742] [General]
- Possibility to select target configurations for run
  - is available in third-party plugin
  - is available in VSCode
- Default values for schematics should be visible in generate dialog
  - is available in third-party plugin
- Searchable list of generators 
  - is available in third-party plugin

In general IMHO the official plugin works somehow unfinished and unstable which throws me back to using the third-party plugins again and again

**Comment by MaxKless** [comment]
Hey @DaSchTour thanks for the write-up, it's really useful to see what isn't working for folks. 

> Possibility to select target configurations for run

This exists on master, just hasn't been released yet. I'll try to release it later today :) 

> Searchable list of generators

Can you expand on what you mean by this? When you run the `Nx Generat (UI)` action, there is a searchable list of generators, right? Is that not what you're looking for?

> Default values for schematics should be visible in generate dialog

Good catch, I just plain hadn't noticed before. That's my bad. Here's a PR fixing it: https://github.com/nrwl/nx-console/pull/1743
